spec/fixtures/pubmed/pubmed_record_29279863.xml

```diff
@@ -0,0 +1,351 @@
+            </Abstract>
+            <AuthorList CompleteYN="Y">
+                <Author ValidYN="Y">
+                    <LastName>Choi</LastName>
+                    <ForeName>Eun Young</ForeName>
+                    <Initials>EY</Initials>
+                    <Identifier Source="ORCID">0000-0003-3226-1486</Identifier>
+                    <AffiliationInfo>
+                        <Affiliation>Department of Pharmacology and Physiology, University of Rochester Medical Center, Rochester, NY 14642.</Affiliation>
+                    </AffiliationInfo>
+                </Author>
+                <Author ValidYN="Y">
+                    <LastName>Ding</LastName>
+                    <ForeName>Song-Lin</ForeName>
+                    <Initials>SL</Initials>
+                    <AffiliationInfo>
+                        <Affiliation>Allen Institute for Brain Science, Seattle, WA 98109.</Affiliation>
+                    </AffiliationInfo>
+                    <AffiliationInfo>
+                        <Affiliation>Institute of Neuroscience, School of Basic Medical Sciences, Guangzhou Medical University, Guangzhou, Guangdong Province 511436, P. R. China.</Affiliation>
+                    </AffiliationInfo>
+                </Author>
+                <Author ValidYN="Y">
+                    <LastName>Haber</LastName>
+                    <ForeName>Suzanne N</ForeName>
+                    <Initials>SN</Initials>
+                    <Identifier Source="ORCID">0000-0002-5237-1941</Identifier>
+                    <AffiliationInfo>
+                        <Affiliation>Department of Pharmacology and Physiology, University of Rochester Medical Center, Rochester, NY 14642.</Affiliation>
+                    </AffiliationInfo>
+                </Author>
+            </AuthorList>
+            <Language>eng</Language>
+            <PublicationTypeList>
+                <PublicationType UI="D016428">Journal Article</PublicationType>
+            </PublicationTypeList>
+            <ArticleDate DateType="Electronic">
+                <Year>2017</Year>
```

Comment:
@peetucket This seems to be a more reliable representation of just the year than the language dependent `YYYY Mon-Mon` field targeted.  Is there a reason this isn't preferred?

Reply:
Seams a better fit.  Note sure if the DateType="Electronic" is important enough to select for, or if just /ArticleDate/Year is safe but I would probably go more broad to avoid missing a year with a different DateType.

Reply:
updated to look in that location and added another location that seems reasonable too -- updated to allow for multiple locations to be specified, with processing stopping at the first that one that appears to be a year